### ISSUE TITLE
Filter Library by "New to watch/Not watched"

### ISSUE THREAD

**Issue by I-NOZex** [Stremio/stremio-features#74]
I think this feature is very useful, i gather all movies and shows that i want to watch or i've watch, and also the tv shows that i follow, and it will be more easily to access them if it was a filter option like that.
IMHO , there is a "Watched" filter that is the opposite and it has less value than this one i'm proposing.
Please considere my request.
I'm also available to help if i can :)
Thank's!

**Comment by jaruba** [comment]
Hi @I-NOZex , I've reviewed your request and also believe that a Library filter for "Not Watched" would be helpful to our users.

Thank you for your suggestion, I will keep you updated to the progress of implementing this feature.

**Comment by I-NOZex** [comment]
Thank you for taking into account my opinion!
Hope that I can use that soon :)
At the time I'm still using the version 3.x, because of issues with 4 on my computer... I'll open a issue btw to explain

**Comment by invu** [comment]
Would be great for discovering movies and shows yourself, kinda like personal recommendations but a lot less effort.

**Comment by maxoku** [comment]
Will it be implemented? It should be easy to add. Just an opposite of "Sort by Watched". Already watched items should be on the bottom, so I wouldn't need to scroll to the bottom to watch unwatched videos. They should be on the top for quick and easy access.

**Comment by djgenesis** [comment]
I also agree that the watched / not watched filtering would be awesome! It's one of the few functionalities currently missing. The current watched filtering is not that helpful. It includes shows I have watched and completed, not watched at all and currently watching. Can't figure out the logic. In a big list in My Library this can be a burden and you have to remember what you have or haven't watched to choose something. Thanks again. Awesome job generally!

**Comment by sayyid5416** [comment]
@jaruba A request made in 2017.... 6 years 👀

**Comment by ensapra** [comment]
Bumping again, how is that going?

**Comment by ensapra** [comment]
I see that the problem is that the Watched sort is not related to actually watched, is technically Watched Time, meaning more time is higher to watched.

Could be nice to actually tag items as watched or not watched and sort them out accordingly instead of arbitrarily selecting if it's watched according to duration*threshold

**Comment by ensapra** [comment]
Bumping!
